charts/kube-prometheus-stack/values.yaml

```diff
@@ -64,6 +64,9 @@ defaultRules:
   ## Annotations for default rules
   annotations: {}
 
+  ## Additional labels for PrometheusRule alerts
+  additionalRuleLabels: {}
+
```

Comment:
> @bmailhe Do we need to add some documentation into the values file and the readme for this feature?

@vsliouniaev vsliouniaev : I added this to the values.yaml. Should I also add a section in the README ?

Reply:
Actually, looks like that requirement is no longer part of the charts. (I'm still living in  helm/charts mentality)